examples/shift-expressions/shift_expressions.bal

```diff
@@ -0,0 +1,42 @@
+import ballerina/io;
+import ballerina/lang.'int as ints;
+
+public function main() {
+    int a = 1;
+    // `<<` performs a left shift. The bits shifted in on the right are `0`.
```

Comment:
The second sentence is not exactly clear. Do you mean that the vacated bit positions are filled using `0`?

Reply:
This again is from the spec. Do we need to mention something like "vacated".

We just shift the bits to the left, adding in `0`s at the right. We don't have to go into details about how shift works right?

Reply:
Yeah, I noticed the definition in the spec later. The thing was, it took me a little while to get what it meant. Isn't there an easier/clearer way to say it?